### PR TITLE
gui-apps/hyprshot: new package

### PR DESCRIPTION
**Pull request by ProMix0**

### CODE REVIEW

**Review by antecrescent — CHANGES_REQUESTED** [review]
Thank you for the contribution!

Your commit is not OpenPGP-signed and missed the `Signed-off-by` tag.
This wikipage contains instructions on how to fix that: https://wiki.gentoo.org/wiki/Project:GURU/Information_for_Contributors#Committing

Hyprshot has optional runtime dependencies on `gui-apps/hyprpicker` and `x11-misc/xdg-user-dirs`. Please add those using the [optfeature](https://devmanual.gentoo.org/eclass-reference/optfeature.eclass/index.html) eclass.

File: gui-apps/hyprshot/hyprshot-1.2.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Copyright 2024 Misha Zavertkin <misha.zavertkin@mail.ru>
+# Distributed under the terms of the ISC License
```
[the remark this answers is not in the document]
Comment (MrRoy) [reply gui-apps/hyprshot/hyprshot-1.2.3.ebuild:2]: This was copied verbatim from wayland-desktop overlay, where the ISC license is enforced

[the remark this answers is not in the document]
Comment (antecrescent) [reply gui-apps/hyprshot/hyprshot-1.2.3.ebuild:2]: Ah, I didn't check that :+1: Thanks for letting me know.

**Comment by ProMix0** [comment]
Thanks for all the review! I had a few questions left

Should I have to send public key to email, specified [here](https://wiki.gentoo.org/wiki/Project:GURU/Information_for_Contributors#Prior_setup)? If so, I haven't @gentoo.org mail cause I'm not a developer. What should I do?

File: gui-apps/hyprshot/hyprshot-1.2.3.ebuild [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# Copyright 2024 Gentoo Authors
+
```
Comment (antecrescent) [inline gui-apps/hyprshot/hyprshot-1.2.3.ebuild:2]: Missing license comment. It was present in the earlier revision.

e.g. `# Distributed under the terms of the GNU General Public License v2`

File: gui-apps/hyprshot/hyprshot-1.3.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# Copyright 2024 Gentoo Authors
+
```
Comment (antecrescent) [inline gui-apps/hyprshot/hyprshot-1.3.0.ebuild:2]: Missing license.

File: gui-apps/hyprshot/hyprshot-9999.ebuild [hunk 4]
```diff
@@ -0,0 +1,31 @@
+# Copyright 2024 Gentoo Authors
+
```
Comment (antecrescent) [inline gui-apps/hyprshot/hyprshot-9999.ebuild:2]: Missing license.

**Review by antecrescent — CHANGES_REQUESTED** [review]

**Comment by antecrescent** [comment]
> Should I have to send public key to email, specified [here](https://wiki.gentoo.org/wiki/Project:GURU/Information_for_Contributors#Prior_setup)?

No. In all cases, you must [create an account](https://bugs.gentoo.org/createaccount.cgi) on Gentoo's Bugzilla with the same email address as the one that's listed in `metadata.xml`, before your contribution can be merged.

If you want to commit regularly, you can request access to the server so that you can push directly to the `dev` branch. To do that, you need to fulfill the requirements listed on https://wiki.gentoo.org/wiki/Project:GURU/Information_for_Contributors#Prior_setup and follow the `file an access request` link to create a new ticket with your information.

**Comment by antecrescent** [comment]
LGTM. Friendly ping @MrRoy since I don't know, if regular contributors such as myself can close GitHub PRs.

**Comment by MrRoy** [comment]
@antecrescent yes you can close PRs via commit tags (`Closes: <URL to PR>`) but you can also use [`app-portage/pram`](https://wiki.gentoo.org/wiki/Gentoo_git_workflow#using_pram) which automates the process

**Pull request closed without merging**